src/modules/settings/index.js

```diff
@@ -146,7 +147,20 @@ class SettingsModule {
         $('body').append(panel);
 
         cdn.get('privacy.html').then(data => $('#bttvPrivacy .tse-content').html(data));
-        cdn.get('changelog.html', true).then(data => $('#bttvChangelog .tse-content').html(data));
+
+        const changelogPanel = $('#bttvChangelog .tse-content');
+        api.get('changelog').then(data => {
+            if (data.status === 200) {
+                changelogPanel.append('<h1>Changelog</h1');
+                data.changelog.forEach(entry => {
+                    const pubdate = window.moment(entry.publishedAt).format('MMM D, YYYY');
+                    changelogPanel.append(`<h2>Version ${entry.version} (${pubdate})</h2> ${entry.body}`);
```

Comment:
2 probably stupid questions
1/ Should body be escaped as well?
2/ Does it matter if the escape happens in the template call or in the template itself (most other examples in the code seem to do it in the template itself)